src/lib/react/index.ts

```diff
@@ -121,6 +122,12 @@ export const reactUniDriver = (containerOrFn: ElementOrElementFinder): UniDriver
 				Simulate.mouseEnter(el);
 			}
 		},
+		pressKey: async (key: KeyType) => {
+			const el = await elem();
+			const def = getDefinitionForKeyType(key);
+			Simulate.keyDown(el, def);
```

Comment:
can we use native events here?
it's better to decouple it from React wherever possible - the only case I found that Simulate was needed was when the element was not attached to the DOM.

Reply:
I think we also need `keyPress`